test/uk/gov/hmrc/nisp/views/QuestionnairFormViewSpec.scala

```diff
@@ -154,7 +154,7 @@ class QuesionnairFormViewSpec extends UnitSpec with MockitoSugar with HtmlSpec w
       assertEqualsMessage(htmlAccountDoc, "article.content__body>form:nth-child(4)>fieldset:nth-child(8)>legend:nth-child(1)" ,"nisp.questionnaire.whatwillyoudonext.question")
     }
     "render page with text  'How could we improve the service?' " in {
-      assertElemetsOwnMessage(htmlAccountDoc, "article.content__body>form:nth-child(4)>fieldset:nth-child(10)>label" ,"nisp.questionnaire.improve.question")
+      assertElemetsOwnMessage(htmlAccountDoc, "article.content__body>form:nth-child(4)>fieldset:nth-child(10)>label" ,"nisp.questionnaire.improve.question" ,"")
```

Comment:
If the value is optional we should use default values or create a different method, let's no throw around blank strings.